### PR TITLE
Add RN 0.60.3 failing build information

### PR DESCRIPTION
**Pull request by krystofwoldrich**
Information for people who encounter this issue https://github.com/getsentry/sentry-react-native/issues/2442

### CODE REVIEW

**Review by marandaneto — APPROVED** [review]

File: src/platforms/react-native/troubleshooting.mdx [hunk 1]
```diff
@@ -234,3 +234,7 @@ _New_:
 ```bash
 shellScript = "set -e\nexport SENTRY_PROPERTIES=sentry.properties\nexport EXTRA_PACKAGER_ARGS=\"--sourcemap-output $DERIVED_FILE_DIR/main.jsbundle.map\"\nWITH_ENVIRONMENT=\"../node_modules/react-native/scripts/xcode/with-environment.sh\"\nREACT_NATIVE_XCODE=\"../node_modules/react-native/scripts/react-native-xcode.sh\"\nSENTRY_CLI_PATH=\"../node_modules/@sentry/cli/bin/sentry-cli\"\n/bin/sh -c \"$WITH_ENVIRONMENT \\\"$SENTRY_CLI_PATH react-native xcode $REACT_NATIVE_XCODE\\\"\"";
 ```
+
+## React Native 0.60.3 failing build
```
Comment (imatwawana) [inline src/platforms/react-native/troubleshooting.mdx:238]:
```suggestion
## React Native 0.60.3 Failing Build
```

File: src/platforms/react-native/troubleshooting.mdx [hunk 2]
```diff
@@ -234,3 +234,7 @@ _New_:
...
+
+React Native 0.60.3 tooling was enexpectively moving the sourcemaps files which caused the Sentry gradle plugin to fail. This was fixed in 0.60.4. For more detailed information follow the [issue](https://github.com/getsentry/sentry-react-native/issues/2442).
```
Comment (imatwawana) [inline src/platforms/react-native/troubleshooting.mdx:240]:
```suggestion
React Native 0.60.3 tooling was unexpectedly moving source maps files, which caused the Sentry Gradle plugin to fail. This was fixed in version 0.60.4. For more detailed information, follow the [issue](https://github.com/getsentry/sentry-react-native/issues/2442).
```

**Review by imatwawana — APPROVED** [review]
A few small fixes

**Comment by krystofwoldrich** [comment]
Thank you @imatwawana

**Pull request merged**